java/src/main/java/vw/VW.java

```diff
@@ -115,6 +115,18 @@ public float predict(String example) {
         }
     }
 
+  public float[] multipredict(String example) {
```

Comment:
Can we change this to conform with the other methods?  By throwning an IllegalStateException the caller is informed about a problem instead of silently succeeding and returning an empty array.  I would suggest something like this

```
    public float[] multipredict(String example) {
        lock.lock();
        try {
            if (isOpen) {
                return multipredict(example, nativePointer);
            }
            throw new IllegalStateException("Already closed.");
        }
        finally {
          lock.unlock();
        }
    }
```